ts/Extensions/Breadcrumbs.ts

```diff
@@ -0,0 +1,1035 @@
+            separatorBBox = separator.getBBox();
+
+            if (!chart.styledMode) {
+                separator.css({
+                    fill: 'white',
+                    stroke: 'black',
+                    'stroke-width': 1
+                });
+            }
+
+            separator.translate(0, separatorBBox.height / 2);
+        }
+
+        return separator;
+    }
+
+    /**
+    * Update.
+    * @function Highcharts.Breadcrumbs#update
+    *
+    * @requires  modules/breadcrumbs
+    *
+    * @param {Highcharts.Breadcrumbs} this
+    *        Breadcrumbs class.
+    * @param {Highcharts.BreadcrumbsOptions} userOptions
+    *        Breadcrumbs class.
+    * @param {boolean} redrawFlag
+    *        Redraw flag
+    */
+    public update(this: Breadcrumbs, userOptions: Highcharts.BreadcrumbsOptions, redrawFlag: boolean = true): void {
+        merge(true, this.options, userOptions);
+
+        if (redrawFlag) {
+            this.destroyGroup();
+            this.redraw();
+        }
+    }
+
+    /**
+    * Update button text when the showFullPath set to false.
+    * @function Highcharts.Breadcrumbs#updateSingleButton
+    *
+    * @requires  modules/breadcrumbs
+    *
+    * @param {Highcharts.Breadcrumbs} this
+    *        Breadcrumbs class.
+    */
+    public updateSingleButton(this: Breadcrumbs): void {
+        const chart = this.chart,
+            breadcrumbsOptions = this.options,
+            breadcrumbsList = this.breadcrumbsList,
+            currentBreadcrumb = breadcrumbsList[this.level],
+            lang = chart.options.lang;
+
+        if (chart.drillUpButton && breadcrumbsOptions && lang) {
+            chart.drillUpButton.attr({
+                text: this.defaultFormatter(currentBreadcrumb)
+            });
+        }
+    }
+
+    /**
+    * Set margins if the floating: false.
+    * @function Highcharts.Breadcrumbs#update
+    *
+    * @requires  modules/breadcrumbs
+    *
+    * @param {Highcharts.Breadcrumbs} this
+    *        Breadcrumbs class.
+    */
+    public setMargins(this: Breadcrumbs): void {
+        const chart = this.chart,
+            breadcrumbsOptions = this.options,
+            defaultBreadcrumheight = 35, // TO DO: calculate that height
+            extraMargin = chart.rangeSelector ? chart.rangeSelector.getHeight() : defaultBreadcrumheight;
+
+        if (
+            breadcrumbsOptions &&
+            !breadcrumbsOptions.floating &&
+            chart.breadcrumbs &&
+            chart.breadcrumbs.breadcrumbsList &&
+            chart.breadcrumbs.breadcrumbsList.length
+        ) {
+            chart.yAxis[0].options.offsets = [extraMargin, 0, 0, 0];
+        } else {
+            chart.yAxis[0].options.offsets = [0, 0, 0, 0];
+        }
+    }
+}
+
+/* eslint-disable no-invalid-this */
+
+if (!H.Breadcrumbs) {
+    H.Breadcrumbs = Breadcrumbs as any;
+
+    addEvent(Chart, 'setOffsets', function (): void {
+        this.breadcrumbs && this.breadcrumbs.setMargins();
```

Comment:
When `floating: false` space is not reserved after the first click. Fix and add a test.